StRoot/StEvent/StFwdTrack.h

```diff
@@ -0,0 +1,101 @@
+/***************************************************************************
+ *
+ * $Id: StFwdTrack.h,v 2.1 2021/01/11 20:25:37 ullrich Exp $
+ *
+ * Author: jdb, Feb 2022
+ ***************************************************************************
+ *
+ * Description: StFwdTrack stores the Forward tracks built from Fst and Ftt
+ *
+ ***************************************************************************
+ *
+ * $Log: StFwdTrack.h,v $
+ * Revision 2.1  2021/01/11 20:25:37  ullrich
+ * Initial Revision
+ *
+ **************************************************************************/
+#ifndef StFwdTrack_hh
+#define StFwdTrack_hh
+
+#include "Stiostream.h"
+#include "StObject.h"
+#include <vector>
+#include "StThreeVectorD.hh"
+#include "StContainers.h"
+
+namespace genfit {
+    class Track;
+}
+
+class StFcsCluster;
+
+struct StFwdTrackProjection {
+    StFwdTrackProjection() {}
+    StFwdTrackProjection( StThreeVectorD xyz, float c[9] ) {
+        XYZ = xyz;
+        memcpy( cov, c, sizeof(cov) );
+    }
+    StThreeVectorD XYZ;
+    float cov[9];
+
+    float dx(){
+        return sqrt( cov[0] );
+    }
+    float dy(){
+        return sqrt( cov[4] );
+    }
+    float dz(){
+        return sqrt( cov[8] );
+    }
+};
+
+class StFwdTrack : public StObject {
+
+public:
+    StFwdTrack( genfit::Track * );
+    vector<StFwdTrackProjection> mProjections;
+    genfit::Track *mGenfitTrack;
```

Comment:
Let's wait for Jason to chime in but I think this dependency contradicts the original concept of StEvent

Reply:
OK, but just a reminder. We discussed the dependency issue in a few S&C meetings, and I implemented this after those discussions

Reply:
Chiming in...  We don't add external package dependencies to StEvent. 

But looking at StFwdTrack.h... other than exposing genfit::Track as public... why do you need to store the genfit track?
It looks like you're just calculating / returning values which can be computed in StEventFiller and saved in member variables of StFwdTrack.